bin/compare-frame

```diff
@@ -121,6 +124,11 @@ def main():
         fraction = np.average(np.abs(diff).ravel() < threshold)
         print(template.format(label=threshold, fraction=fraction))
 
+    # compare resolution
+    print('(resolution_a, resolution_b):')
+    isclose = np.average(np.isclose(a['resolution'], b['resolution']))
```

Comment:
Is average the best way to determine if the test should pass or fail? Would it be more concrete to focus on a particular sigma bin, 0.01 for example? That way the test could report that agreement was accurate to a specific level (and fail if it is not).

Reply:
It's probably not the best, but it's simple. I've been using this script to generate a quick summary of the difference between two output files. 

This is computing the fraction of pixels that are within np.isclose (it's not computing the average difference between pixels). A test could require that the fraction pixels in agreement needs to be above some threshold like 99%. Then the question is what is the right way to to define "agreement". Some of the checks performed on `flux` and `ivar` are like what you are suggesting but I don't know what a similar "pull" distribution would look like for `resolution`.

The unit tests that compare the resolution are performed at the patch extraction level in [py/gpu_specter/test/test_ex2d_patch.py](https://github.com/sbailey/gpu_specter/blob/master/py/gpu_specter/test/test_ex2d_patch.py).